main.go

```diff
@@ -33,9 +33,8 @@ func main() {
 	filterPath := ""
 	flaggy.String(&filterPath, "f", "filter", "Path to filter on in `git log -- <path>`. When in filter mode, the commits, reflog, and stash are filtered based on the given path, and some operations are restricted")
 
-	dump := ""
-	flaggy.AddPositionalValue(&dump, "gitargs", 1, false, "Todo file")
-	flaggy.DefaultParser.PositionalFlags[0].Hidden = true
+	gitArgs := ""
```

Comment:
When naming variables, we should take care to use singular or plural forms only when appropriate. If we're only storing a single scalar value (as is the case here) we should use the singular form i.e. `gitArg`.